tests/baselines/reference/tsbuild/sample1/incremental-declaration-changes/listEmittedFiles.js

```diff
@@ -0,0 +1,209 @@
+//// [/lib/incremental-declaration-changesOutput.txt]

```

Comment:
This is the output from running the below command when the change is made.

Reply:
So are there other filenames that are _not_ being printed because only their timestamp changed?

Reply:
Correct